projects/demo/src/modules/components/editor-new/editor-new.component.ts

```diff
@@ -52,26 +38,34 @@ export class ExampleEditorNewComponent extends AbstractExampleTuiControl {
         '!!raw-loader!./examples/import/editor-options-token.md'
     );
 
-    readonly example1: FrontEndExample = {
```

Comment:
unfortunately, it will make editor's tests flaky(

Reply:
Probably, I should try to make this PR great again)))
https://github.com/Tinkoff/taiga-ui/pull/1504

Reply:
Run several times, works